sfcasts/crud-controller.md

```diff
@@ -1,171 +1,90 @@
+
+So what's going on?
+
+Click "Questions" and look at the URL. It starts with `/admin` and then a bunch of query parameters. Turns out, everything EasyAdmin is handled via a single giant route. It all runs through the `DashboardController` route - the `/route` that's above the index. So when we go to the `QuestionCrudController`, it's actually matching this route here with extra query parameters to say which CRUD controller and which action runs. You can see `crudController` and `crudAction` right there.
+
+We're rendering `QuestionCrudController`, but in the context of our `DashboardController`. When we go to this page, in order to get the CRUD config, EasyAdmin first calls `configureCrud` on our dashboard. Then it calls `configureCrud` on our specific ride controllers. In this case, `QuestionCrudController` is super powerful. It means that we can override things on a dashboard level, like set some CRUD settings to apply to all sections or for one specific `crudController`.
+
+I'll prove it! Go back to `AbstractDashboardController`. In `configureCrud`, every area has four pages. I'll hold "cmd" and click to open this `Crud` thing. There's some constants on top here. Every CRUD section has an index page. That's this in 'edit' page, a 'new' page, and also a 'detail' page. Each page can then refer to a set of actions. These are links or buttons. For example, on the index page, right now, we have an action for editing and an action for deleting. We *could* add more actions or remove those actions.
+
+You can see how this is configured down in `configureActions`. This is for the dashboard, so this is the action configuration that applies to all of our sections. You can see that for the index page. It adds new "Edit" and "Delete" actions for the detail page. There's `EDIT`, `INDEX`, and `DELETE`. And if you're on the edit page, you have the actions `SAVE_AND_RETURN` and `SAVE_AND_CONTINUE`. One thing you'll notice if you look closely here is that we *do* have a detail page, but nobody actually links to it. You don't see an action calling `DETAIL` on any of these pages. So the page exists, but it's not really used out of the box. So let's add that. 
+
+We'll go to `DashboardController` and it doesn't matter where but I'll go down to the bottom. I'm going to go to "Code Generate" or "cmd + N" on a Mac, and go to "Override Methods..."" and override `configureCrud`. The first thing we want to do is call `paren::configureCrud`. So it can instantiate that object.
```

Comment:
Please tell me it is "cmd + N" and not "M". I'll die inside if I've misunderstood that the entire time. x.x

Reply:
I left the "..." at the end of Override Methods since it actually has those there in the video, but I can totally cut them out if they're not necessary.

Reply:
Okay, and here's another little nugget I wondered about. *Sometimes* you leave mistakes in the tutorial instead of taking them out. I wasn't sure if this was one of those times or not. Where you override `configureCrud` and then in the next paragraph, walk it back to `configureActions`. I re-wrote it to account for the mistake, should it be included for whatever reason, but I should easily be able to remove it and re-word things if it's not sticking around. If I'm taking this out, then please ignore the weird wording of the next paragraph. I'll fix it.